### PR TITLE
Move all imports for d2l-tabs to single file

### PR DESCRIPTION
**Pull request by rgoessl**

### CODE REVIEW

File: web-components/d2l-tab-panel.js [hunk 1]
```diff
@@ -1 +0,0 @@
-import 'd2l-tabs/d2l-tab-panel.js';
```
Comment (dlockhart) [inline web-components/d2l-tab-panel.js]: I can't tell from the diff... can this file just be deleted?

Comment (rgoessl) [reply web-components/d2l-tab-panel.js]: The import is now in `d2l-tabs.js`, don't need that file anymore. Is that not what you meant by putting them in the same bsi import?

Comment (rgoessl) [reply web-components/d2l-tab-panel.js]: The file was deleted if that's what you're asking

Comment (dlockhart) [reply web-components/d2l-tab-panel.js]: Haha yeah sorry was just asking if the file was deleted -- now I see where in the diff it says that. All good!

**Pull request merged**